src/Services/Message/index.js

```diff
@@ -15,9 +16,26 @@ const createSystemMessage = (text) => {
   return message;
 };
 
+const addMessageToMessages = async (id, text, user) => {
+  let room = await RoomService.findRoom(id);
+
+  if(room.messages.length > 100 ){
+    room.messages.shift();
+  }
+
+  const message = createMessage(text, user);
+  message.isSystemMessage = false;
+
+  room.messages.push(message);
+
+  await room.save();
+  return true;
```

Comment:
```suggestion
  return message;
```
ileride belki kaydedilen mesajın `id` gibi bilgileri bize lazım olabilir bundan dolayı kaydedilen msajı dönmemizi tavsiye ederim.